ms/MSSel/MSSelection.cc

```diff
@@ -808,44 +808,44 @@ namespace casacore { //# NAMESPACE CASACORE - BEGIN
       {
       case ANTENNA_EXPR:
 	{
-	  if (overRide  ||  MSAntennaParse::thisMSAErrorHandler.null()) {
+	  if (overRide  ||  !MSAntennaParse::thisMSAErrorHandler) {
             if (mssEH == NULL) {
-              MSAntennaParse::thisMSAErrorHandler = mssEH;
+              MSAntennaParse::thisMSAErrorHandler.reset(mssEH);
```

Comment:
Since `mssEH` is NULL, a reset without arguments is more clear:
```suggestion
              MSAntennaParse::thisMSAErrorHandler.reset();
```

Reply:
good catch